packages/firestore/src/local/shared_client_state.ts

```diff
@@ -173,6 +176,7 @@ export interface SharedClientState {
  * encoded as part of the key.
  */
 interface MutationMetadataSchema {
+  snapshotVersionMicros: number;
```

Comment:
We should store this as seconds/nanos.

Reply:
No longer applicable since we don't need to look these version up from LocalStorage.